.github/workflows/build.yaml

```diff
@@ -36,6 +65,16 @@ jobs:
           restore-keys: Library-
       - if: matrix.targetPlatform == 'Android'
         uses: jlumbroso/free-disk-space@v1.3.1
+      - name: Download plugins
+        uses: actions/download-artifact@v4
+        with:
+          name: plugins-ubuntu-latest
+          path: Assets/Plugins
+      - name: Extract plugins
+        run: |
+          cd Assets/Plugins
+          tar -xzvf plugins.tar.gz
```

Comment:
You'd need to extract for all platforms - otherwise only Ubuntu gets linked

Reply:
Fixed in https://github.com/PisterLab/micromissiles-unity/pull/40/commits/2456fec2cfe76486b477e3390184cf13b31e2418